library/kani_macros/src/sysroot/contracts/check.rs

```diff
@@ -60,7 +60,8 @@ impl<'a> ContractConditionsHandler<'a> {
                 let wrapper_args = if let Some(wrapper_call_args) =
                     inner.iter_mut().find_map(|stmt| try_as_wrapper_call_args(stmt, &wrapper_name))
                 {
-                    let wrapper_args = make_wrapper_args(wrapper_call_args.len(), attr.len());
+                    let wrapper_args =
+                        make_wrapper_idents(wrapper_call_args.len(), attr.len(), "_wrapper_arg_");
```

Comment:
Since `_wrapper_arg_` is used multiple times, please add a `const` string for it.

Reply:
Implemented as suggested.